tests/src/test/java/alluxio/client/rest/S3ClientRestApiTest.java

```diff
@@ -100,74 +116,143 @@ public void before() throws Exception {
   }
 
   @Test
-  public void putBucket() throws Exception {
-    final String bucket = "bucket";
-    createBucketRestCall(bucket);
-    // Verify the directory is created for the new bucket.
-    AlluxioURI uri = new AlluxioURI(AlluxioURI.SEPARATOR + bucket);
-    Assert.assertTrue(mFileSystemMaster
-        .listStatus(uri, ListStatusContext.defaults()).isEmpty());
+  public void listAllMyBuckets() throws Exception {
+    mFileSystem.createDirectory(new AlluxioURI("/bucket0"));
+    SetAttributePOptions setAttributeOptions =
+        SetAttributePOptions.newBuilder().setOwner("user0").build();
```

Comment:
I would like to not set owner like this separately but instead have a file system that does this properly for me. (because I'm going to need it to correct various creation apis with authorization). How do I do that?

Reply:
Can we use `AuthenticatedUserRule` for this purpose?

Reply:
Does it matter that the cluster is using `.setProperty(PropertyKey.SECURITY_AUTHENTICATION_TYPE, AuthType.NOSASL.getAuthName())`? It may have to be changed to SIMPLE.